src/addBusinessDays/test.ts

```diff
@@ -118,27 +160,63 @@ describe('addBusinessDays', function () {
       })
 
       // Then we expect to have the working Saturdays ignored
-      assert.deepStrictEqual(result, new Date(2022, 1, 22))
+      assert.deepStrictEqual(result, new Date(2022, 1 /* Feb */, 22))
+    })
+
+    it('can correctly add days when ending on a false exception', function () {
+      // Given business days of Monday - Friday
+      // And an exception on a Friday
+
+      // When we add 5 business days and end on the exception
+      const result = addBusinessDays(new Date(2022, 1 /* Feb */, 4), 5, {
+        exceptions: { '02/11/22': false },
+      })
+
+      // Then we expect to have the disabled Friday ignored
+      assert.deepStrictEqual(result, new Date(2022, 1 /* Feb */, 14))
     })
 
-    it('can override business days with exceptions over weekends', function () {
+    it('can correctly add days when starting on a false exception', function () {
+      // Given business days of Monday - Friday
+      // And an exception on a Friday
+
+      // When we start on the exception and add 6 business days
+      const result = addBusinessDays(new Date(2022, 1 /* Feb */, 4), 6, {
+        exceptions: { '02/04/22': false },
+      })
+
+      // Then we expect to have the disabled Friday ignored
+      assert.deepStrictEqual(result, new Date(2022, 1 /* Feb */, 14))
+    })
+
+    it('can correctly add days when starting and ending on a false exception', function () {
```

Comment:
@chr1sjf0x this is the test that is failing, which theoretically shouldn't happen because our date picker won't allow it... so is it even a useful test?

Reply:
this passes now